src/components/button/Button.js

```diff
@@ -6,6 +6,11 @@ import { UITextBody, UITextDisplay, UITextSmall } from '../typography';
 import cx from 'classnames';
 import theme from './theme.css';
 
+export const BUTTON_LEVELS = {
```

Comment:
Since you moved them here now, it would be nice to add the other levels as well of course 😅.

And maybe already use it inside the component where possible (even if's just for the prop-types).

Reply:
Following up!